types/constants.ts

```diff
@@ -1,6 +1,11 @@
-import { BlockExplorer, NetworkId, NetworkName, NativeToken } from "./handler";
+import { BlockExplorer, NetworkId, NetworkName, NativeToken, Rpc } from "./handler";
 import { CHAINS_IDS, EXTRA_RPCS, NETWORK_CURRENCIES, NETWORK_EXPLORERS, NETWORK_FAUCETS } from "./dynamic";
 
+export declare const chainIDList: Record<string, string>;
```

Comment:
these two declared consts and the `ChainId` type can be removed

Reply:
Good catch. I have removed them.